wrappers/services_test.go

```diff
@@ -447,6 +453,15 @@ ListenStream=%s
 
 `, filepath.Join(s.tempdir, "/var/snap/hello-snap/12/sock2.socket"))
 	c.Check(sock2File, testutil.FileContains, expected)
+
+	expected = fmt.Sprintf(
+		`[Socket]
+Service=snap.hello-snap.svc1.service
+FileDescriptorName=sock3
+ListenStream=%s
+
+`, filepath.Join(s.tempdir, "/run/user", strconv.FormatUint(uint64(sys.Geteuid()), 10), "snap.hello-snap/sock3.socket"))
```

Comment:
This is similar to above (and the diff will change this part too): when reading this its slightly misleading as it may make the reader think the value is dependent on the runtime uid. But its strictly dependent on the user configured in the service yaml. So I would prefer here to not use sys.Geteuid() as well.

Reply:
@mvo5 I updated code per proposal. Initially I was trying to avoid hardcoding uid but you are right that it actually made it worse and code was giving wrong impression.
And true, this will anyway need to be revisited when we land support for user services